### PR TITLE
Created antisnake

### PR DESCRIPTION
**Pull request by olya-yevsieienko**
- [DEMO LINK](https://olya-yevsieienko.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://olya-yevsieienko.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

**Review by AntonYegorov — APPROVED** [review]

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+.box {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  &__block {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    min-width: 300px;
+    height: 300px;
```
Comment (Olexandr6) [inline src/styles/main.scss:32]: Let's create a var for `300px` value

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+  font-family: "Arial", Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
 body {
   margin: 0;
 }
+
+.box {
```
Comment (Olexandr6) [inline src/styles/main.scss:23]: [Checklist](https://github.com/mate-academy/layout_antisnake/blob/master/checklist.md) 4th point

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,50 @@
+$colors:
+  "1" rgb(100%, 0%, 0%),
+  "2" rgb(80%, 0%, 0%),
+  "3" rgb(60%, 0%, 0%),
+  "4" rgb(40%, 0%, 0%),
+  "5" rgb(20%, 0%, 0%),
+  "6" rgb(0%, 0%, 0%);
```
Comment (Olexandr6) [inline src/styles/main.scss:7]: Create a separate file for vars

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="box">
+      <div class="box__block box__block--1"></div>
```
Comment (Olexandr6) [inline src/index.html:12]: Let's don't create additional classes `box__block--n`, use `:nth-child` pseudo-class instead

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Almos done🔥

File: src/styles/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,9 @@
+$block-height: 300px;
+
+$colors:
+  "1" rgb(100%, 0%, 0%),
+  "2" rgb(80%, 0%, 0%),
+  "3" rgb(60%, 0%, 0%),
+  "4" rgb(40%, 0%, 0%),
+  "5" rgb(20%, 0%, 0%),
+  "6" rgb(0%, 0%, 0%);
```
Comment (jstmpelowycz) [inline src/styles/_variables.scss:9]: 🔥

**Review by jstmpelowycz — APPROVED** [review]
Nice!

**Pull request closed without merging**